### DISCUSSION TITLE
Certificate Page Not Found

### DISCUSSION THREAD

**Discussion by Time4Pizza** [sandboxie-plus/Sandboxie#3553] [General]
Hello. I am trying to update my supporter certificate so I can use Sanboxie in ISO mode, but when I click on the link to support/certificate page it comes up with a 404 error. 

Perhaps the website is just being updated? Bad timing by me if that is the case. However, if there is new link or some other method to get a supporter certificate please let me know. Thank you so much for Sandboxie, it's a lifesaver for the project I'm working on!

**Comment by bastik-1001** [comment]
There appear to be issues with the web server, as the main website of xanasoft.com can be reached, but subdomains appear to inaccessible. The main page shows on older state. The forum is not reachable. Even sandboxie-plus.com is not showing what it is supposed to do.

Edit: The weblate instance can be reached, what I find odd.

**Comment by DavidXanatos** [comment]
it seams an update to direct admin has reset some config should be fixed very soon.

**Reply by Time4Pizza** [reply]
Awesome! Thank you for letting me know. I am in no rush, just happy to know I'll have capabilities back at some point soon.

**Comment by DavidXanatos** [comment]
should work again

**Comment by Time4Pizza** [comment]
Thank you sir. Got a fraud alert, but I approved the transaction. Run it again if it declined the first time.   From: DavidXanatosSent: Tuesday, January 9, 2024 1:47 PMTo: sandboxie-plus/SandboxieCc: Time4Pizza; AuthorSubject: Re: [sandboxie-plus/Sandboxie] Certificate Page Not Found (Discussion #3553) should work again—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>
